athena/data/feature_normalizer.py

```diff
@@ -50,32 +85,43 @@ def apply_cmvn(self, feat_data, speaker):
         feat_data = (feat_data - mean) / tf.sqrt(var)
         return feat_data
 
-    def compute_cmvn(self, entries, speakers, featurizer, feature_dim):
+    def compute_cmvn(self, entries, speakers, featurizer, feature_dim, is_parallel=False):
         """ Compute cmvn for filtered entries """
+
         start = time.time()
         for tar_speaker in speakers:
             logging.info("processing %s" % tar_speaker)
             initial_mean = tf.Variable(tf.zeros([feature_dim], dtype=tf.float32))
             initial_var = tf.Variable(tf.zeros([feature_dim], dtype=tf.float32))
             total_num = tf.Variable(0, dtype=tf.int32)
```

Comment:
i suggest those three variables should be created only when we use parallel mode

Reply:
done in the latest commit